docs/ kpoboe.md

```diff
@@ -0,0 +1,3 @@
+- [ ] **NAME** Kpoboe
+- [ ] **Occupation** CMtoolsAdmin
+- [ ] **Location** DC
```

Comment:
Is this Washington DC?  I think that's what you mean, but I'm not sure.  :see_no_evil: